ember-headlessui/addon/components/disclosure/panel.js

```diff
@@ -0,0 +1,10 @@
+import Component from '@glimmer/component';
+
+import { tracked } from '@glimmer/tracking';
+import { guidFor } from '@ember/object/internals';
+
+export default class Panel extends Component {
+  panelGuid = `${guidFor(this)}-tailwindui-disclosure-panel`;
+  
+  @tracked isOpen = false;  
```

Comment:
why does this have its own isOpen state when the parent component also has isOpen?